kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -888,6 +887,22 @@ def count_objects_by_tags(self, metric, scraper_config):
         for tags, count in iteritems(object_counter):
             self.gauge(metric_name, count, tags=list(tags))
 
+    def kube_deployment_count(self, metric, scraper_config):
+        config = self.object_count_params[metric.name]
+        metric_name = "{}.{}".format(scraper_config['namespace'], config['metric_name'])
+        seen = set()
+
+        for sample in metric.samples:
+            tags = []
+            namespace = self._label_to_tag("namespace", sample[self.SAMPLE_LABELS], scraper_config)
+            deployment = self._label_to_tag("deployment", sample[self.SAMPLE_LABELS], scraper_config)
```

Comment:
```suggestion
            deployment = sample[self.SAMPLE_LABELS].get("deployment")
```
as you won't be using this as a tag, this saves a bit of processing

Reply:
ah, nice! 🙇‍♂️